src/stores/presets.js

```
@@ -0,0 +1,542 @@
+                    failedSet.add(repository);
+                    return null;
+                }),
+            ),
+        );
+
+        failedRepositoryNames.value = Array.from(failedSet).map((repository) => repository.name);
+        repositories.value = nextRepositories.filter((repository) => !failedSet.has(repository));
+
+        try {
+            await confirmSourceVersions();
+            repositories.value.forEach((repository) =>
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)